addon/components/g-recaptcha.js

```diff
@@ -60,14 +63,18 @@ export default Component.extend({
     }
   },
 
+  getScript(src) {
+    let scriptTag = document.createElement('script');
+    scriptTag.src = src;
+    document.body.appendChild(scriptTag);
+  },
 
   // Lifecycle Hooks
 
   didInsertElement() {
     this._super(...arguments);
-    next(() => {
-      this.renderReCaptcha();
-    });
+    window.__emberGRecaptcha__onLoadCallback = () => { this.renderReCaptcha();  }
```

Comment:
@nbrookie Why such double underscores in `__emberGRecaptcha__onLoadCallback`? Is there some naming convention supporting such decision?